packages/hardware-trezor/src/transformers/txIn.ts

```diff
@@ -8,11 +8,21 @@ import { resolvePaymentKeyPathForTxIn } from './keyPaths';
  * Transforms the given Cardano input transaction to the Trezor
  * input transaction format using the given trezor input resolver.
  */
-export const toTrezorTxIn: Transform<Cardano.TxIn, Promise<trezor.CardanoInput>, TrezorTxTransformerContext> = async (
+export const toTrezorTxIn: Transform<Cardano.TxIn, Promise<Trezor.CardanoInput>, TrezorTxTransformerContext> = async (
   txIn,
   context?
 ) => ({
   path: (await resolvePaymentKeyPathForTxIn(txIn, context)) ?? undefined,
-  prev_hash: Buffer.from(txIn.txId).toString('hex'),
+  prev_hash: txIn.txId,
```

Comment:
I'm not sure this is correct

Reply:
I suggest applying [this change](https://github.com/input-output-hk/cardano-js-sdk/pull/734#discussion_r1213916043) and then running a test that signs any tx with TrezorKeyAgent. If this is not correct then it'll throw

Reply:
@mkazlauskas @tomislavhoracek im getting this error in the "unit tests" for trezor in the "wallet" package, could this be related?
<img width="931" alt="grafik" src="https://github.com/input-output-hk/cardano-js-sdk/assets/172414/1164a673-3298-4439-8785-21dcf154f765">

Reply:
hm ok no, getting the same error when switching back to the old format